File: solutions/c/two-fer/2/src/two_fer.c
#include "two_fer.h"
#include <string.h>
#include <stdio.h>

#define BUFFER_SIZE (100)

void two_fer(char *buffer, const char *name)
{
    sprintf(buffer, "One for %s, one for me.", name == NULL ? "you" : name);
}

File: solutions/python/two-fer/2/two_fer.py
def two_fer(name: str = "you") -> str:
    """
    Return a string following the 'two-fer' phrase pattern.

    If a name is provided, returns "One for <name>, one for me."
    If no name is given, defaults to "you".

    Args:
        name: The name to include in the phrase. Defaults to "you".

    Returns:
        A string in the format "One for <name>, one for me."
    """
    return f"One for {name}, one for me."
